peachjam/models/core_document_model.py

```diff
@@ -755,6 +755,16 @@ def extension(self):
         return os.path.splitext(self.filename)[1].replace(".", "")
 
 
+class ArticleAttachment(AttachmentAbstractModel):
```

Comment:
Is it not possible to put this in article.py?

Reply:
@longhotsummer unfortunately no, we get circular import errors. Is there a good way to handle circular imports in our modules? it keeps coming up.

Reply:
Hmm, depends on the context. Suggestions:

* put abstract models into a separate file so that they can be imported everywhere
* use strings to refer to other models (like you have done below)
* local imports